textpipe/doc.py

```diff
@@ -509,3 +507,51 @@ def aggregate_word_vectors(self, model_name=None, aggregation='mean', normalize=
             return numpy.var(vectors, axis=0).tolist()
         else:
             raise NotImplementedError(f'Aggregation method {aggregation} is not implemented.')
+
+    @functools.lru_cache()
```

Comment:
LRU-caching this on top of saving it in a dict seems like overkill. Not sure if the LRU-caching is needed here.